.github/scripts/buildActions.sh

```diff
@@ -3,8 +3,15 @@
 # Used to precompile all Github Action node.js scripts using ncc.
 # This bundles them with their dependencies into a single executable node.js script.
 
+# This function is a polyfill for GNU's realpath that (hopefully) works on (almost) any shell
+# By default, realpath is not available on macOS w/o first installing coreutils. :(
+realpath_polyfill() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+
 # In order for this script to be safely run from anywhere, we cannot use the raw relative path '../actions'
```

Comment:
I'm curious actually why we need this? Can we just assume that the user is running from root via the `npm run` command?

Reply:
Yeah, not super critical. I just nixed it, and the script still works if you run `npm run gh-actions-build`